tests/vcd_disk.py

```diff
@@ -87,8 +87,54 @@ def test_008_update_disk(self):
         assert task.get('status') == TaskStatus.SUCCESS.value
 
         print ("Updating VDC disk: " + str(etree.tostring(task, pretty_print=True), "utf-8"))
- 
 
+    def test_008_update_disk_iops(self):
```

Comment:
change the test number to something unique

Reply:
How does the test number matter here ? The test names are unique already.